src/app/features/tech-record/components/vehicle-technical-record/vehicle-technical-record.component.ts

```diff
@@ -73,6 +73,10 @@ export class VehicleTechnicalRecordComponent implements OnInit {
     return StatusCodes;
   }
 
+  hasPlates(techRecord: TechRecordModel) {
+    return techRecord.plates!.length > 0 ?? false;
```

Comment:
This will break when plates are `undefined`.
```suggestion
    return (techRecord.plates?.length ?? 0) > 0;
```